### ISSUE TITLE
getting-started guide is outdated?

### ISSUE THREAD

**Issue by jonua** [modelmapper/modelmapper.github.io#2]
Hi,

now, in getting started guide in section "Expicit mapping" [placed](https://github.com/modelmapper/modelmapper.github.io/blame/master/getting-started/index.md#L107-L112) example:

```
modelMapper.addMappings(mapper -> {
  mapper.map(src -> src.getBillingAddress().getStreet(),
      Destination::setBillingStreet);
  mapper.map(src -> src.getBillingAddress().getCity(),
      Destination::setBillingCity);
});
```

But in current latest version (1.1.2) `org.modelmapper.PropertyMap` is not a functional interface. Example from modelmapper repo is [actual](https://github.com/modelmapper/modelmapper/blob/master/examples/src/main/java/org/modelmapper/gettingstarted/GettingStartedExample.java#L34)

Please, fix your documentation

**Comment by chhsiao90** [comment]
The functional interface is supported by `org.modelmapper.ExpressionMap` [ExpressionMap](https://github.com/modelmapper/modelmapper/blob/master/core/src/main/java/org/modelmapper/ExpressionMap.java).

And it looks like we have wrong example, I will update it.

```
modelMapper.typeMap(Order.class, OrderDto).addMappings(
  mapper -> {
    ...
  });
```

**Comment by chhsiao90** [comment]
Thanks for reporting the issue, issue fixed in 10a7d967.
